src/main/java/meteordevelopment/meteorclient/MeteorClient.java

```diff
@@ -182,6 +183,10 @@ private void onOpenScreen(OpenScreenEvent event) {
         }
 
         wasWidgetScreen = event.screen instanceof WidgetScreen;
+
+        if (!GuiThemes.get().showHUDInEditor()) return;
+        if (!(event.screen instanceof HudEditorScreen)) return;
+        mc.options.hudHidden = false;
```

Comment:
guard statements here would make it annoying to add onto this event handler, id rather this be a
```java
if (GuiThemes.get().showHUDInEditor() && event.screen instanceof HudEditorScreen) {
    mc.options.hudHidden = false;
}
```